src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -121,6 +121,18 @@ def tab(id, text, opts)
     content_tag('li', link_to(h(text), opts), link_opts)
   end
 
+  def webui2_tab(id, text, opts)
```

Comment:
I would be very glad if we could get rid of these superfluous helpers that just add complexity while being rather inflexible.